lit_tests/kernel/wave/codegen.py

```diff
@@ -305,37 +305,19 @@ def mma(
         # CHECK: %[[R3:.+]] = arith.addi %[[R2]], %[[R1]] : index
         # CHECK: %[[R4:.+]] = vector.load %0[%[[R3]], %[[C0]]] : memref<64x16xf16, strided<[16, 1], offset: ?>>, vector<4xf16>
         # CHECK: %[[ALLOC:.+]] = memref.alloc() : memref<32x16xf16, #gpu.address_space<workgroup>>
-        # CHECK: %[[R5:.+]] = arith.muli %[[WG0]], %[[C32]] : index
-        # CHECK: %[[R6:.+]] = arith.divsi %[[TX]], %[[C4]] : index
-        # CHECK: %[[R7:.+]] = arith.addi %[[R6]], %[[R5]] : index
-        # CHECK: vector.store %4, %[[ALLOC]][%[[R7]], %[[C0]]] : memref<32x16xf16, #gpu.address_space<workgroup>>, vector<4xf16>
-        # CHECK: %[[R8:.+]] = arith.muli %[[WG0]], %[[C32]] : index
-        # CHECK: %[[R9:.+]] = arith.divsi %[[TX]], %[[C4]] : index
-        # CHECK: %[[R10:.+]] = arith.addi %[[R9]], %[[R8]] : index
-        # CHECK: %[[R11:.+]] = vector.load %[[ALLOC]][%[[R10]], %[[C0]]] : memref<32x16xf16, #gpu.address_space<workgroup>>, vector<4xf16>
+        # CHECK: vector.store %4, %[[ALLOC]][%[[R3]], %[[C0]]] : memref<32x16xf16, #gpu.address_space<workgroup>>, vector<4xf16>
```

Comment:
Just wondering, should this %4 be %[[R11]] ?

Reply:
ahhh, right, good catch, thanks! :)